pwiz_tools/Shared/ProteowizardWrapper/MsDataFileImpl.cs

```diff
@@ -612,19 +612,59 @@ public double[] GetTotalIonCurrent()
             }
             using (var chromatogram = ChromatogramList.chromatogram(0, true))
             {
-                if (chromatogram == null)
+                return chromatogram?.getIntensityArray()?.data.Storage();
+            }
+        }
+
+        public class QcTrace
+        {
+            public QcTrace(Chromatogram c)
+            {
+                Name = c.id;
+                if (Name.ToLowerInvariant().Contains(@"pressure"))
                 {
-                    return null;
+                    MeasuredQuality = "pressure";

```

Comment:
I prefer to see text like this expressed as constants to avoid the same text used in multiple places, requiring you type it the same in both.

Reply:
When the CV is updated to include PSI as a unit (https://github.com/bio-ontology-research-group/unit-ontology/issues/20) I plan to switch these to enumerated values instead of strings. It won't hurt to use string constants in the meantime though.